src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public enum Colors {
+    red,

```

Comment:
```suggestion
public enum Color {
    red,
```
- better use singular for enums
- all enum values better write in upper case because they are the constants